logic/block_reward.js

```diff
@@ -33,13 +33,13 @@ const __private = {};
 class BlockReward {
 	constructor() {
 		// Array of milestones
-		this.milestones = constants.REWARDS.MILESTONES;
+		this.milestones = REWARDS.MILESTONES;
 
 		// Distance between each milestone
-		this.distance = Math.floor(constants.REWARDS.DISTANCE);
+		this.distance = Math.floor(REWARDS.DISTANCE);
```

Comment:
Why we need to call `Math.floor` on this?

Reply:
@4miners I can't comment on this. Is there any chance that rewards constants can be float?

Reply:
@ManuGowda can you comment on this ?

Reply:
They cannot.

Reply:
@4miners then having `this.distance` variable doesn't make sense either.

Reply:
Yep, I was referencing to all of them.

Reply:
you can directly use `REWARDS.DISTANCE` directly. I don't know the intention behind `floor` logic here, but as Mariusz said rewards will not be a float.

Reply:
@4miners Let's handle this in a separate issue. see: https://github.com/LiskHQ/lisk/issues/2412